app/models/case/sar/offender.rb

```diff
@@ -40,6 +41,8 @@ def searchable_fields_and_ranks
                  recipient: :string,
                  reply_method: :string,
                  subject_address: :string,
+                 request_dated: :date,
+                 requester_reference: :string,
```

Comment:
Apologies again! We have duplicate key in here for `requester_reference` - duplicated on line 51 below. Puma complains about it on server startup. Maybe we could order the keys alphabetically again - seems to be have got out of sync a little